test/integration/connection-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const Connection = require('../../src/api/common/connection');
+
+const request1 = {
+  command: 'server_info'
+};
+
+const request2 = {
+  command: 'account_info',
+  account: 'r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59'
+};
+
+const request3 = {
+  command: 'account_info'
+};
+
+const request4 = {
+  command: 'account_info',
+  account: 'invalid'
+};
+
+function makeRequest(connection, request) {
+  return connection.request(request).then((response) => {
+    console.log(request);
+    console.log(JSON.stringify(response, null, 2));
+  }).catch((error) => {
+    console.log(request);
+    console.log(error);
+  });
+}
+
+function main() {
+  const connection = new Connection('wss://s1.ripple.com');
```

Comment:
Live test against public rippled network? Looks like this is the objective of /integration tests?